posts/2017-10-15-creating-a-serverless-graphql-to-rest-gateway-on-top-of-the-flickr-api.md

```diff
@@ -749,15 +794,19 @@ async function getPhotos(
 
 export default getPhotos
```

Comment:
I'll have to make some minor edits to this code block. Updating my eslint rules yesterday revealed to me that I'm improperly using async/await here.  It's unnecessarily wrapping the function in a generator, when the function will return a promise regardless since await is being used in the return statement. Removing these shaved a bunch of bytes off my application bundle.